sdks/node/usage-guide-voice.md

```diff
@@ -0,0 +1,94 @@
+# Usage and Migration Guide for Node.js Voice 2.x
+
+Bandwidth's Node.js Voice SDK has seen significant changes when moving from 1.x to 2.x. Outlined below are a number of common scenarios where you'll see changes.
+
+---
+
+## Client Initialization
+
+### 2.x
+
+```js
+import { Client, ApiController } from '@bandwidth/voice';
+
+const client = new Client({
+    basicAuthPassword: 'password',
+    basicAuthUserName: 'username'
+});
+
+const controller = new ApiController(client);
+```
+
+### 1.x
+
+```js
+const BandwidthVoice = require('@bandwidth/voice');
+BandwidthVoice.Configuration.basicAuthUserName = "username";
+BandwidthVoice.Configuration.basicAuthPassword = "password";
+const controller = BandwidthVoice.APIController;
+
+const BandwidthBxml = require('@bandwidth/bxml');
+```
+
+---
+
+## Create Phone Call
+
+### 2.x
+
+```js
+const response = await controller.createCall('account-id-123', {
```

Comment:
Lots of users get hung up on what the accountId is - can we normalize this to be accountId123 - without dashes - so that no one gets confused and throws their application ID in there

Reply:
Are you just referring to update the previous to this?
```suggestion
const response = await controller.createCall('accountId123', {
```

Reply:
Yes - i think we should format the dummy variables the same way that the actual values will be formatted